pylint/checkers/design_analysis.py

```diff
@@ -171,13 +171,18 @@ def __init__(self, linter=None):
         self.stats = None
         self._returns = None
         self._branches = None
-        self._stmts = 0
+        self._stmts = None
 
     def open(self):
         """initialize visit variables"""
         self.stats = self.linter.add_stats()
         self._returns = []
         self._branches = defaultdict(int)
+        self._stmts = []
+
+    def _inc_all_stmts(self, amount):
```

Comment:
`_increment_all_statements`?

Reply:
I tried to keep it consistent with the other variable names in this file - the _inc_ and the _stmts are there as well. I'd prefer that name but it wouldn't match the other names in this file.